examples/tutorials/04_linear_models_example.ipynb

```diff
@@ -24,17 +24,21 @@
     "import warnings\n",
```

Comment:
<p>Move this TF / Torch difference above model config definition. And describe more thoroughly why need them in the first place</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>